src/integration/kotlin/SnapshotSenderIntegrationTest.kt

```diff
@@ -73,11 +73,11 @@ class SnapshotSenderIntegrationTest : StringSpec() {
         "Verify for every source collection a finished file was written to s3" {
             logger.info("Checking $bucketUri ...")
 
-            //s3 in:  test/output/db.core.addressDeclaration-000001.txt.bz2.enc
-            //s3 out: test/status/db.core.addressDeclaration-000001.txt.bz2.enc.finished
+            //s3 in:  test/output/db.core.addressDeclaration-000001.txt.gz.enc
```

Comment:
can we just remove the commented out stuff?

Reply:
They are actual comments not commented out code.